src/Administration/Controller/AdministrationController.php

```diff
@@ -291,6 +291,20 @@ public function sanitizeHtml(Request $request, Context $context): JsonResponse
         );
     }
 
+    #[Route(path: '/api/_admin/config-parameter/{parameterName}', name: 'api.admin.config-parameter', methods: ['GET'])]
+    public function getConfigParameter(string $parameterName): JsonResponse
+    {
+        $allowedParameters = [
```

Comment:
IMO we can add this parameter into `index` method so we don't need to introduce a new public API and another request in the admin